environment.yml

```diff
@@ -4,62 +4,75 @@ channels:
   - defaults
 
 dependencies:
-  - armadillo==8.200.0
-  - blas==1.1
-  - bullet==2.86.1
-  - bz2file==0.98
-  - bzip2==1.0.6
-  - cmake>=3.10
-  - cspice==66
-  - curl==7.60.0
-  - doxygen==1.8.14
-  - eigen==3.3.3
-  - embree==2.14.0
-  - geos==3.5.1
-  - geotiff==1.4.2
-  - gmm==5.0
-  - gmp==6.1.2
-  - gsl==2.2.1
-  - hdf5==1.8.18
-  - icu==58.2
-  - jama==125
+  - armadillo
+  - blas
+  - bullet
+  - bz2file
+  - bzip2
+  - cspice
+  - curl
+  - doxygen
+  - eigen
+  - embree
+  - geos
+  - geotiff
+  - gmm
+  - gmp
+  - gsl
+  - hdf5
+  - icu
+  - jama
   - jpeg==9b
   - kakadu==1
-  - krb5==1.14.2
+  - krb5
   - libpng>=1.6.34
-  - libprotobuf==3.5.2
-  - libtiff>=4.0.9
-  - libxml2==2.9.7
+  - libprotobuf
+  - libtiff
+  - libxml2
   - make
-  - mesalib==17.2.0
-  - mysql==5.7.20
-  - mysql-connector-c==6.1.6
-  - nanoflann==1.2.2
-  - ninja==1.7.2
+  - mesalib
+  - mysql
+  - mysql-connector-c
+  - nanoflann
+  - ninja
   - conda-forge/label/gcc7::nn
-  - openblas==0.2.19
-  - opencv
-  - openssl==1.0.2n
-  - pcl==1.8.1
-  - pip==9.0.1
-  - protobuf==3.5.2
-  - python==3.6
-  - qhull==7.2.0=0
+  - opencv=3.3.0
+  - openssl
+  - pcl
+  - pip
+  - protobuf>=3.5.2
+  - python
+  - qhull
   - qt=5.9.6
   - qwt=6.1.3
-  - setuptools=38.5.1
-  - sip==4.18
-  - sqlite==3.13.0
-  - suitesparse==4.5.4
-  - superlu==5.2.1
-  - tnt==126=0
-  - wheel==0.30.0
-  - x264==20131218
-  - xalan-c==1.11
-  - xerces-c==3.1.4
-  - xorg-kbproto==1.0.7
+  - setuptools
+  - sip
+  - sqlite
+  - suitesparse
+  - superlu
+  - tnt
+  - wheel
+  - x264
+  - xalan-c
+  - xerces-c
+  - xorg-kbproto
   - xorg-libice
   - xorg-libsm
-  - xorg-libx11==1.6.4
+  - xorg-libx11
   - xorg-libxi
-  - zlib==1.2.11
+  - zlib
+
+  - cmake>=3.10
+  - eigen
+  - jupyter
```

Comment:
Ale dependencies, originally because current Ale release isn't up to date, theoretically just having Ale would install the same things but you will have to replace the Conda installed Ale with a local version of Ale to actually use it in Isis.